src/codec/RowReaderV1.cpp

```diff
@@ -219,7 +219,7 @@ Value RowReaderV1::getValueByIndex(const int64_t index) const noexcept {
     case PropertyType::STRING:
       return getString(index);
     default:
-      LOG(ERROR) << "Unknown type: " << apache::thrift::util::enumNameSafe(vType);
+      LOG(FATAL) << "Unknown type: " << apache::thrift::util::enumNameSafe(vType);
```

Comment:
why ？ This will coredump.

Reply:
BTW, should `LOG(FATAL`) appear as little as possible in the code?  
Because LOG(FATAL) directly leads to service coredump

Reply:
Actually for v1, there won't be other types, I think we could fail only in this case. I am ok if use `LOG(ERROR)` here.

Reply:
Fixed